_layouts/page.html

```diff
@@ -6,7 +6,7 @@
     <div class="container">
         <div class="logo">
             <a href="/">
-                <img src="/assets/img/logo.svg"/>
+                <img class="w-100" src="/assets/img/mlh-prep.png"/>
```

Comment:
@sohamsshah makes a good point about accessibility. Here is a good resource for [alt text](https://webaim.org/techniques/alttext/) and a well thought-out post on [StackOverflow](https://stackoverflow.com/questions/23526791/when-is-it-acceptable-to-leave-the-alt-text-empty-on-an-html-image-if-ever) if you would like to learn more about it. Note that the alt text doesn't need to be overly descriptive; something like "mlh prep program logo" would be sufficient.